README.md

```diff
@@ -96,11 +96,10 @@ The bundled shared object is stored in directory
 
 ## Unix
 
-On Unix, the `yaz-config` utility is used to get compiler flags and
+On Unix, the `pkg-config` utility is used to get compiler flags and
```

Comment:
Is this now true on all version of Unix, or only on Debian GNU/Linux? Do we need to include both possible sets of instructions?

Reply:
Good question. It's part of all RPM/Deb today. yaz.pc.in was created in 2007.

Reply:
OK, but I'm not clear on what you plan to do. You're saying (I think) that RPM uses the same `pkg-config` system now as Debian does, but are we confident that's true of all Linux distributions? And of non-Linux Unixes? And of non-Unix operating systems?

In other words, it seems to me that we still need to support, and document, yaz-config as well as pkg-config. Am I missing something?

Reply:
With YAZ, `make install` installs yaz.pc. It's not RPM/Deb specific.  But I agree that on Mac (macports?) that wouldn't fly.